swix/res/Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest.swr

```diff
@@ -0,0 +1,14 @@
+use vs
+
+package name=Microsoft.VisualStudio.VC.Ide.TestAdapterForGoogleTest
+        version=$(Version)
+        vs.package.type=vsix
+        vs.package.installSize=22265856
+        vs.package.vsixId=GoogleTestAdapterVSIX.8486b54e-5768-482b-b224-7d069ca509d8
+
+vs.payloads
+  vs.payload source=$(VsixOutDir)\Packaging.TAfGT.vsix
```

Comment:
Is duplicating this in other swr intentional?

Reply:
This isn't built right now as well. I wanted to keep the structure, maybe should have dumped some empty files instead but this will make it easier going forward.